packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -878,6 +878,26 @@ class _LargeTitleNavigationBarSliverDelegate
     if (!transitionBetweenRoutes || !_isTransitionable(context)) {
       return navBar;
     }
+    
```

Comment:
881, 882, 886, and 895 are extra lines you should delete as described in [Linux analyze] failure.